packages/db/lib/start.mjs

```diff
@@ -19,7 +19,10 @@ export async function start (_args) {
   }))
 
   const { configManager } = await loadConfig({
-    string: ['to']
+    string: ['to', 'hot-reload'],
+    alias: {
+      hotReload: ['hot-reload']
+    }
```

Comment:
Are there other places in the CLI where a long option name is being aliased to another long option name? If not, I'd suggest sticking with one naming convention for long option names i.e. `hyphen-separated`.

Reply:
for example, there is  `allowEnv: ['allow-env', 'E']`
left side is just for mapping variable for the code use

Reply:
Got it! Thanks for explaining that @salesh 👌🏻